webrtc-c/canary/src/Config.cpp

```diff
@@ -2,29 +2,78 @@
 
 namespace Canary {
 
-STATUS mustenv(CHAR const* pKey, Config::Value<const CHAR*>* pResult)
+STATUS Config::init(INT32 argc, PCHAR argv[])
+{
+    // TODO: Probably also support command line args to fill the config
+    STATUS retStatus = STATUS_SUCCESS;
+
+    CHK(argv != NULL, STATUS_NULL_ARG);
+    MEMSET(this, 0, SIZEOF(Config));
+
+    if (argc == 2) {
+        DLOGI("Reading configuration from %s\n", argv[1]);
+        CHK_STATUS(initWithJSON(argv[1]));
+    }
+
+    CHK_STATUS(initWithEnvVars());
```

Comment:
Perhaps move initWithEnvVars before the initWithJSON

Reply:
I think the idea is to use JSON as the main values, and use env vars as a fallback. So, the order should be correct as it is. Is there any reason why it should be swapped?

Reply:
You can either do the non-overriding version of the configs and call JSON and then ENV initialization or simply do the overrides in which case env vars should be first and then JSON. I am OK either way - both approaches are OK.

Reply:
I think I'll leave it is as it is since the code is already written. We can update this in the future when needed.